messages/en.json

```
@@ -1,4 +1,5 @@
 {
-  "admin/test": "test",
-  "store/test": "test"
+  "store/listPrice.default": "Original price: {listPrice}",
```

I think is better to use kebab case instead of camel case

```suggestion
  "store/list-price.default": "Original price: {listPrice}",
```